docs/pulseload.rst

```diff
@@ -10,19 +10,9 @@ posting your own data.
 Configuration
 -------------
 
-Specify the exchanges to read from using environment variables. For example::
-
-    PULSE_DATA_INGESTION_SOURCES = [
-        {
-            "exchange": "exchange/taskcluster-treeherder/v1/jobs",
-            "destinations": [
-                'tc-treeherder'
-            ],
-            "projects": [
-                'mozilla-inbound.#'
-            ]
-        }
-    ]
+If you don't want all the sources provided by default in ``settings.py``, you can specify the exchange, the projects, or destinations to read from using an environment variable in your vagrant shell. A working example::
+
```

Comment:
Given the env example below uses `#` for its destination and projects, should we say above, something like `A working example which...` and say what it is reading from? Is it production and inbound?

Reply:
Yeah, I like this!  :)